src/components/common/formCheckBoxField.jsx

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from "react";
+import { Field } from "react-final-form";
+
+class FormCheckBox extends Component {
```

Comment:
V Reacte sa komponenty daju vytvorit 2 sposobmi, a v pripade, ze komponenta nepotrebuje drzat state interne, da sa jednoduchsie zapisat ako functional component (tie pomocou hook-ov mozu obsahovat state tiez, takze class componenty vo vacsine pripadov uz nie je treba - samozrejme, neznamena to, ze sa nemozu pouzit). Osobne ich preferujem viac, kedze na to staci menej kodu.

Priklad:

```jsx
const FormCheckBox = () => (
  <div>
    <label>{this.props.label}</label>
    <Field name={this.props.name} component="input" type="checkbox" />
  </div>
);
```